pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -105,36 +104,12 @@ set -o nounset # fail if variables are unset
 set -o xtrace
 
 OSD_ID="$ROOK_OSD_ID"
-CEPH_FSID=%s
 OSD_UUID=%s
 OSD_STORE_FLAG="%s"
 OSD_DATA_DIR=/var/lib/ceph/osd/ceph-"$OSD_ID"
 CV_MODE=%s
 DEVICE="$%s"
 
-# "ceph.conf" must have the "fsid" global configuration to activate encrypted OSDs
```

Comment:
@satoru-takeuchi This basically reverts #10333 since these comments indicate we can remove this when Quincy support is removed. Please review.

Reply:
@travisn We can remove this workaround. This problem is addressed in [ceph#46588](https://github.com/ceph/ceph/pull/46588) and this fix is included in [ceph v18.2.0](https://github.com/ceph/ceph/commits/v18.2.0/src/ceph-volume/ceph_volume/configuration.py).